kolibri/utils/logger.py

```diff
@@ -234,6 +236,14 @@ def get_default_logging_config(LOG_ROOT, debug=False, debug_database=False):
                 "formatter": "simple_date",
                 "encoding": "utf-8",
             },
+            "requests": {
```

Comment:
Hrm - was there a need to add an additional handler here? My concern is that creating more log files makes it more complicated for users to upload logs for bug reports.

Reply:
I wanted to log requests into a separate file which I could achieve only through an additional handler. I am not sure if there is any other way to achieve that.

Reply:
I would like to point out that this issue requires us to log requests in a separate file when they get logged.

Reply:
To quote the issue:
> I'd like the option of a) turning off request logging, or b) with reqeust logging enabled, writing them to a file that's distinct from the file used for application logging.

Solving a) is sufficient for closing the issue. Currently you are also solving b), but not in an optional way. We could go all in and add an option for where the requests log should optionally be redirected to, but that seems more complicated than necessary to meet the primary need here.

The reason for my pushback here is that we should not be changing the default behaviour here, as the issue is only about adding optional configuration.

Reply:
Okay. For now I'll stick to solving a). But I am curious for b) as well